src/client/app/effects/github.ts

```diff
@@ -134,12 +141,17 @@ export class GitHubEffects {
                 new Snippet.UpdateInfoAction({ id: snippetId, gist: gist.id, gistOwnerId: gist.owner.login })])
         )
         .catch(exception => {
+            let { gistOwnerId, id } = exception;
+            let action = (gistOwnerId && id)
+            ? new Snippet.UpdateInfoAction({id: id, gistOwnerId: gistOwnerId})
```

Comment:
Indent these?  And I think the usual convention for this is for the "?" to be at the beginning of the previous line, as is the case with ":"